### DISCUSSION TITLE
[SOLVED] problems viewing tickets from another user

### DISCUSSION THREAD

**Discussion by MixTecnico** [YetiForceCompany/YetiForceCRM#14726] [How to]
good guys I have a problem that must be silly but I can't find a solution .... in the incident ticket system the main user creates these incidents, but for a month the other users have not seen the tickets created, the users. They are administrators too and the permissions are in the same group and role as the main user, any ideas?

# 🐞 bug report

### 👷 Testing

- [ ] I confirm that I tested it at https://gitstable.yetiforce.com
- [ ] I confirm that I tested it at https://gitdeveloper.yetiforce.com

### 🌍 Your Environment

| Environment                         | Version / Name |
| ----------------------------------- | -------------- |
| YetiForce                           |
| Web server (name and version)       |                |
| PHP                                 |                |
| Browser (name and version)          |                |
| Operating System (name and version) |                |
| Database (name and version)         |                |
| Service Pack version                |                |

**Comment by rskrzypczak** [comment]
If aminaistrotor does not see entries, you must have messed up the code yourself.
This is not a bug in the base application, so let's move on to the discussion.

**Reply by MixTecnico** [reply]
The whole system works perfectly, except for this inconvenience, I had a server migration but I did not see any problem but until now, do you know what I could see to solve it?

**Reply by MixTecnico** [reply]
It is as if the user had a filter to view only the month 04, they do not see anything else

**Reply by MixTecnico** [reply]
SOLVED when changing the server there was a change in directories since now a custom php profile is being used through PHP-FPM, thanks also for your answer, I leave the solution in case it is useful for someone

**Reply by drakub** [reply]
@MixTecnico can you provide me with this solution. I am facing a similar issue here
